src/components/allarticlespage/AllArticlesPage.js

```diff
@@ -1,50 +1,104 @@
 import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { toastr } from 'react-redux-toastr';
 import { CardList } from '../Card';
-import mockArticles from '../../../mockdata/articles';
+import Pagination from '../common/Pagination';
 import PopularPosts from '../home/PopularPosts';
+import { fetchCategory } from '../../actions/articleActions';
+
+export class AllArticlesPage extends React.Component {
```

Comment:
You should import Component from React like we have always done, and not use React.Component

Reply:
@KvNGCzA feedback well received.
@KvNGCzA done.